.aspect/rules/external_repository_action_cache/npm_translate_lock_LTE4Nzc1MDcwNjU=

```diff
@@ -2,26 +2,26 @@
 # Input hashes for repository rule npm_translate_lock(name = "npm", pnpm_lock = "@@//:pnpm-lock.yaml").
 # This file should be checked into version control along with the pnpm-lock.yaml file.
 .npmrc=-2065072158
-pnpm-lock.yaml=-481966245
+pnpm-lock.yaml=-1309835144
 examples/npm_deps/patches/meaning-of-life@1.0.0-pnpm.patch=-442666336
 package.json=-275319675
 pnpm-workspace.yaml=-1178830835
 examples/js_binary/package.json=-41174383
+examples/linked_empty_node_modules/package.json=-1039372825
 examples/macro/package.json=857146175
 examples/npm_deps/package.json=-1377141392
 examples/npm_package/libs/lib_a/package.json=-1377103079
 examples/npm_package/packages/pkg_a/package.json=1006424040
 examples/npm_package/packages/pkg_b/package.json=1041247977
 examples/webpack_cli/package.json=1911342006
 js/private/coverage/bundle/package.json=-1543718929
+js/private/image/package.json=-1260474848
```

Comment:
Why did the order of these change? 🤔

Reply:
Not sure

Reply:
I'm going to try this, once your pr mergess: https://github.com/aspect-build/rules_js/pull/1796

Reply:
It looks like it is sorted now